subsystem/utils_test.go

```diff
@@ -27,6 +27,13 @@ func strToUUID(s string) *strfmt.UUID {
 	return &u
 }
 
+func registerNode(ctx context.Context, clusterID strfmt.UUID, name string) *models.Host {
```

Comment:
Why duplicate code it it's already exist in other tests? 
If you define a generic function then please fix the other tests to use it as well and put it in utils_test.go

Reply:
Fixed.